dr_ensenso/src/opencv.cpp

```diff
@@ -19,23 +41,41 @@ cv::Mat toCvMat(NxLibItem const & item, std::string const & what) {
 }
 
 cv::Mat toCameraMatrix(NxLibItem const & item, std::string const & camera, std::string const & what) {
-	int error = 0;
-	cv::Mat result = cv::Mat::zeros(3, 3, CV_64F);
-	for (std::size_t i=0; i<3; i++) {
-		for (std::size_t j=0; j<3; j++) {
-			result.at<double>(i,j) = item[itmMonocular][camera == "Left" ? itmLeft : itmRight][itmCamera][j][i].asDouble(&error);
-			if (error) throw NxError(item, error, what);
-		}
+	cv::Mat result;
+	try {
+		result = cameraMatrixImpl(item[itmCalibration][itmMonocular][camera == "Left" ? itmLeft : itmRight], what);
+	} catch (NxError const & e) {
+		throw e;
```

Comment:
This try/catch seems rather unnecessary.

Reply:
Fair enough